### PR TITLE
🚀 enhance: Remove expected earning calculation from product listing and editing pages

### PR DESCRIPTION
**Pull request by devAsadNur**
### All Submissions:

* [x] My code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)
* [x] My code satisfies feature requirements
* [x] My code is tested
* [x] My code passes the PHPCS tests
* [ ] My code has proper inline documentation
* [ ] I've included developer documentation (optional)
* [x] I've added proper labels to this pull request

### Changes proposed in this Pull Request:
Removed expected earning calculation from product listing and editing pages.

Closes # .

### How to test the changes in this Pull Request:

1.
2.
3.

### Changelog entry

> enhance: Remove expected earning calculation from product listing and editing pages

### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.

### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.

* [ ] Correct — Does the change do what it’s supposed to? ie: code 100% fulfilling the requirements?
* [ ] Secure — Would a nefarious party find some way to exploit this change? ie: everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities?
* [ ] Readable — Will your future self be able to understand this change months down the road?
* [ ] Elegant — Does the change fit aesthetically within the overall style and architecture?

### CODE REVIEW

File: assets/src/js/product-editor.js [hunk 1]
```diff
@@ -1002,49 +1002,6 @@
...
-            let commission = $('span.vendor-earning').attr( 'data-commission' );
-            let product_id = $( 'span.vendor-earning' ).attr( 'data-product-id' );
-            let product_price = $( 'input.dokan-product-regular-price' ).val();
-            let sale_price = $( 'input.dokan-product-sales-price' ).val();
-            let earning_suggestion = $('.simple-product span.vendor-price');
-
-            earning_suggestion.html( dokan.i18n_calculating );
-
-            $.get( dokan.ajaxurl, {
-                action: 'get_vendor_earning',
```
Comment (Aunshon) [inline assets/src/js/product-editor.js:1015]: Also need to remove the ajax action hook and callback in the following php file.

* [includes/Ajax.php #L56](https://github.com/weDevsOfficial/dokan/blob/develop/includes/Ajax.php#L156)

**Review by Aunshon — CHANGES_REQUESTED** [review]
Need some changes

**Review by Aunshon — APPROVED** [review]
